hooks/publish.py

```diff
@@ -962,7 +992,7 @@ def _get_publish_linked_entity_name(self, task_settings, item):
         publisher = self.parent
 
         # Start with the item's fields
-        fields = copy.copy(item.properties.get("fields", {}))
+        fields = item.properties.get("fields", {})
```

Comment:
We should not be mutating `fields` here. IMHO, fields should be immutable after Collection and any mutation should be done on a local copy.